src/nxdoc/nuxeo-server/data-store/indexing-and-query/nxql.md

```diff
@@ -995,6 +995,7 @@ Where:
     *   [simple_query_string](http://www.elastic.co/guide/en/elasticsearch/reference/5.6/query-dsl-simple-query-string-query.html)
     *   [regex](http://www.elastic.co/guide/en/elasticsearch/reference/5.6/query-dsl-regexp-query.html)
     *   [wildcard](http://www.elastic.co/guide/en/elasticsearch/reference/5.6/query-dsl-wildcard-query.html)
+    *   [more_like_this](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/query-dsl-mlt-query.html)
```

Comment:
:+1: 
Hints could be sorted in alphabetical order (almost the case before).

Reply:
Done! The Geo Location has been moved up even if it has a sublevel